anypoint-mq/3.x/modules/ROOT/pages/anypoint-mq-consume.adoc

```diff
@@ -52,17 +52,25 @@ For information about acknowledgment timeouts, see xref:anypoint-mq-ack.adoc#ack
 
 == Empty Queue Management
 
-When a queue has messages, the Consume operation returns immediately with one of the messages. When a queue is empty, the operation behavior depends on how you set the 'pollingTime' parameter:
+When a queue has messages, the Consume operation returns immediately with one of the messages. When a queue is empty, the operation behavior depends on the value of the `pollingTime` parameter.
+
+* No value
++
+By default, the Consume operation waits for 10000 milliseconds (10 seconds) for a new message to arrive when consuming from an empty queue. Consume fails if no message arrives in that timeframe.
 
 * `0`
 +
-If you set the `pollingTime` parameter to `0`, the Consume operation returns immediately without any error and produces a `null` message as the result when the queue is empty.
+If the value of `pollingTime` is `0`, the Consume operation waits for the default of 10000 milliseconds (10 seconds).
```

Comment:
Should we keep this part? "produces a `null` message as the result when the queue is empty."
When pollingTime = 0, we don't throw an error like the other scenarios here.

Reply:
Yes, I don't know why I deleted it.